frontend/src/components/regions/DueTimeFormModal/index.jsx

```diff
@@ -34,10 +53,15 @@ function DueTimeFormModalUnconnected({
 const mapStateToProps = (state) => {
   return {
     ...state,
+    latestApiCallStatus: getLatestMatchingCall({
+      callLog: state.CARD_SET_DUE_TIME,
+      requestData: {},
+    }),
     cardId: state.DueTimeFormModal.cardId || null,
     users: state.apiEntities.user || {},
     card: state.apiEntities.cards || {},
```

Comment:
I just realized that `card` should be `cards` since it contains all known cards. Can you please fix that? Remember to fix it in the props as well